ChegeBryan/politico
# #163681838 Transfers UI files

### What does this PR do?

Incorporates UI files to develop branch

#### Description of Task to be completed?


- [x] Merge gh-pages to develop.

#### How should this be manually tested?
View project at `https://github.com/ChegeBryan/politico`. click on the GitHub compare option, set `develop` branch as your `base` branch and `gh-pages` as your `compare`. The out window should display `There isn’t anything to compare.`

#### What are the relevant pivotal tracker stories?
[#163681838](https://www.pivotaltracker.com/story/show/163681838)
